crates/nu-command/src/network/url/build_query.rs

```diff
@@ -42,6 +42,11 @@ impl Command for SubCommand {
                 example: r#"{a:"AT&T", b: "AT T"} | url build-query"#,
                 result: Some(Value::test_string("a=AT%26T&b=AT+T")),
             },
+            Example {
+                description: "Outputs a query string representing the contents of this record",
+                example: r#"{a[]: ["one", "two"], b: "three"} | url build-query"#,
```

Comment:
I am not sure if the `a[]` may be a bit confusing here. If you read it it might imply you need to append it to be able to use the list form but it will work with any key. (even if the consuming API may have some ideas about what it permits there)

Reply:
Fair point. Changed.